app/pb_kits/playbook/pb_currency/_currency.html.erb

```diff
@@ -1,13 +1,14 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-
+    class: object.classname,
+    aria: object.aria) do %>
+  
   <%= object.label %>
   <div class="pb_currency_wrapper">
-    <%= object.dollar_sign %>
-    <%= object.value %>
-    <%= object.unit %>
+    <%= pb_rails("body", props: { color: "light", text: object.unit }) %>
```

Comment:
Looks like the `body` before was passed a block. You could do that here in the view. OR, you could use the `children` prop that @xjunior added in #326 and pass it inline. --- If this already looks how it needs too, disregard this comment :).